src/dataGenerator.js

```diff
@@ -131,6 +131,20 @@ module.exports = async ({
                     });
                 } else {
                     if (format == 'avro') {
+                        if (!avroSchemas[topic]) {
```

Comment:
I’m curious why this is necessary. Wouldn’t all the avroSchemas entries have been populated during iteration 0 above?

Reply:
It does not seem to be the case for the relational data

Reply:
Actually managed to fix this! The problem was with the `!dryRun` check.

Reply:
Ha!

*shakes fist at javascript*

Reply:
I think that the problem is with the way that the `debug` arg is passed :D it is just a string rather than a bool hence causing that problem 😅

Reply:
Oh, duh! Nice catch